Source/WebCore/style/StyleScope.cpp

```diff
@@ -338,20 +338,25 @@ Vector<Ref<ProcessingInstruction>> Scope::collectXSLTransforms()
 }
 #endif
 
-void Scope::collectActiveStyleSheets(Vector<RefPtr<StyleSheet>>& sheets)
+auto Scope::collectActiveStyleSheets() -> ActiveStyleSheetCollection
```

Comment:
Do we now prefer this style in WebKit? I thought that it was an ugly exception only needed in rare template programming situations, not a generally preferred style.

Reply:
No that we prefer but this avoids having to repeat `Style::Scope::`.

Reply:
It's not uncommon in WebKit to use this to avoid repeating the nested class name, i.e. to avoid writing this as `Scope::ActiveStyleSheetcollection Scope::collectActiveStyleSheets()`.